integration/e2e/e2e_test.go

```diff
@@ -914,3 +924,42 @@ func hashFile(file string) string {
 func chaincodeContainerNameFilter(n *nwo.Network, chaincode nwo.Chaincode) string {
 	return fmt.Sprintf("^/%s-.*-%s-%s$", n.NetworkID, chaincode.Label, hashFile(chaincode.PackageFile))
 }
+
+func channelParticipationList(n *nwo.Network, o *nwo.Orderer, expectedChannels []string, systemChannel ...string) {
+	authClient, unauthClient := OrdererOperationalClients(n, o)
+	listChannelsURL := fmt.Sprintf("https://127.0.0.1:%d/participation/v1/channels", n.OrdererPort(o, nwo.OperationsPort))
+
+	expectedChannelList := &types.ChannelList{}
+
+	channelToChannelInfoShort := func(channel string) types.ChannelInfoShort {
+		return types.ChannelInfoShort{
+			Name: channel,
+			URL:  fmt.Sprintf("/participation/v1/channels/%s", channel),
```

Comment:
Use the constant from:
https://github.com/hyperledger/fabric/blob/946016f7f415459447d1e77cf2f3f65eb9d4b78a/orderer/common/channelparticipation/restapi.go#L31

Reply:
I don't think we should use the constant here. It will help us ensure we don't unintentionally make any breaking changes to the API.

Reply:
Sound good.